src/network-schemas.js

```diff
@@ -72,15 +72,33 @@ function registerNetworkSchemas() {
     template: "#video-template",
     components: [
       {
-        component: "position"
+        component: "position",
+        requiresNetworkUpdate: vectorRequiresUpdate(0.001)
       },
       {
-        component: "rotation"
+        component: "rotation",
+        requiresNetworkUpdate: vectorRequiresUpdate(0.5)
       },
       "visible"
     ]
   });
 
+  NAF.schemas.add({
+    template: "#interactable-template",
```

Comment:
I think this was replaced in master -- do we need this?

Reply:
This got added back in intentionally, but will get removed again once the environments get updated (doing that now)